app/Http/Controllers/ExportController.php

```diff
@@ -432,41 +508,33 @@ private function createCsv($rows, $headers) {
             fputcsv($csv, $row);
         }
         rewind($csv);
-        $csv = stream_get_contents($csv);
 
-        return $csv;
+        return stream_get_contents($csv);
     }
 
     /**
      * Get a CSV or HTML of the export
      *
-     * @var array  $csv      The data.
+     * @var mixed  $csv      The data.
      * @var string $title
      * @var string $fileType 'csv' or 'html'
      *
-     * @return The thing to present to the user.
+     * @return mixed
      */
-    private function getExport($csv, $title, $fileType) {
-        if ($fileType == self::CSV) {
+    private function getExport($csv, string $title, string $fileType) {
+        if ($fileType === self::CSV || $fileType === self::JSON) {
             return response($csv)
                 ->withHeaders([
                     'Content-Type'        => 'text/csv',
                     'Cache-Control'       => 'no-store, no-cache',
-                    'Content-Disposition' => 'attachment; filename="' . slug($title) . '.csv"',
-                ]);
-        } else if ($fileType == self::JSON) {
-            return response($csv)
-                ->withHeaders([
-                    'Content-Type'        => 'text/json',

```

Comment:
Use `text/csv` for both json and CSV? That doesn't cause problems?

Reply:
It works as intended but since it looks kinda weird I updated it to use text/csv or text/json accordingly